components/server/src/setup.rs

```diff
@@ -211,11 +241,13 @@ pub fn initial_metric(cfg: &MetricConfig) {
 #[allow(dead_code)]
 pub fn overwrite_config_with_cmd_args(config: &mut TiKvConfig, matches: &ArgMatches<'_>) {
     if let Some(level) = matches.value_of("log-level") {
-        config.log_level = logger::get_level_by_string(level).unwrap();
+        config.log.level = logger::get_level_by_string(level).unwrap();
+        config.log_level = slog::Level::Info;
```

Comment:
How about keeping these parameters changed for backward compatibility?

Reply:
If you mean to set it to the default value, I think it's better to write ``TiKvConfig::default().log_level` here?